docs/sketches.rst

```diff
@@ -0,0 +1,48 @@
+***************
```

Comment:
I'd be inclined to move this to another PR. I'm a bit worried about setting the `make_sketch` API in stone...

Reply:
Good idea